flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -37,8 +37,8 @@ under the License.
 
 	<!-- Allow users to pass custom connector versions -->
 	<properties>
-		<cassandra.version>2.2.5</cassandra.version>
-		<driver.version>3.0.0</driver.version>
+		<cassandra.version>3.11.3</cassandra.version>
```

Comment:
when you upgrade the dependency's version, please also provide it's dependency tree about the two version.

Reply:
done :)

Reply:
@yanghua it is ok now ? :)